fluss-server/src/main/java/org/apache/fluss/server/replica/Replica.java

```diff
@@ -246,10 +249,36 @@ public Replica(
     private void registerMetrics() {
         // get root metric in the reference: bucket -> table -> tabletServer
         TabletServerMetricGroup serverMetrics =
-                bucketMetricGroup.getTableMetricGroup().getServerMetricGroup();
+                ((TableMetricGroup) bucketMetricGroup.getParentMetricGroup())
```

Comment:
This is still very hack. We can introduce a private `SimpleBucketMetricGroup` in `CoordinatorMetricGroup` like `SimpleTableMetricGroup`. And expose it as `MetricGroup` for method `getTableBucketMetricGroup`.